modules/db/src/main/scala/gem/dao/StaticConfigDao.scala

```diff
@@ -67,77 +64,172 @@ object StaticConfigDao {
     }
   }
 
+  /** Combines lower-level GMOS statements into higher-level inserts and
+    * selects.
+    */
+  private object Gmos {
+
+    import gem.config.Gmos.GmosNodAndShuffle
+    import gem.enum.Instrument.{ GmosN, GmosS }
+    import StaticConfig.{ GmosNorth, GmosSouth }
+
+    def insertNorth(sid: Int, gn: GmosNorth): ConnectionIO[Int] =
+      for {
+        i <- insertNodAndShuffle(sid, GmosN, gn.common.nodAndShuffle)
+        j <- Statements.Gmos.insertNorth(sid, gn).run
+      } yield i + j
```

Comment:
It's not really clear what we should return from updates that affect many tables. The value doesn't really mean anything so it might be better as `Unit`.